src/HotChocolate/Spatial/src/Types.Spatial/GeoJSONPositionScalar.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections;
+using HotChocolate.Language;
+using NetTopologySuite.Geometries;
+
+namespace HotChocolate.Types.Spatial
+{
+    public class GeoJSONPositionScalar : ScalarType<Coordinate>
+    {
+        /// https://tools.ietf.org/html/rfc7946#section-3.1.1
+        public GeoJSONPositionScalar() : base("Position")
+        {
+            // TODO : move to resources.
+            Description = "A position is an array of numbers. There MUST be two or more elements.";
+        }
+
+        public override bool IsInstanceOfType(IValueNode literal)
```

Comment:
I think we have to rework this as well ... but we should I think first discuss how strict instance of should be .... @PascalSenn should also I think look at the scalar type interface and may be introduce a TryParseLiteral etc....